### ISSUE TITLE
Six Flags

### ISSUE THREAD

**Issue by GMVargas** [glasklart/hd#7977]
**App Name:** Six Flags
**Bundle ID:** com.sixflags.ios
**iTunes ID:** <a target="_blank" href="http://getart.dreamnet.at?id=988953433">988953433</a>
**iTunes URL:** <a target="_blank" href="https://itunes.apple.com/us/app/six-flags/id988953433?mt=8&uo=4">https://itunes.apple.com/us/app/six-flags/id988953433?mt=8&uo=4</a>
**App Version:** 2.0.0
**Seller:** Six Flags Entertainment Corporation
**Developer:** <a target="_blank" href=https://itunes.apple.com/us/developer/six-flags-entertainment-corporation/id988953432?uo=4>© Six Flags Entertainment Corporation</a>
**Supported Devices:** iPad2Wifi, iPad23G, iPhone4S, iPadThirdGen, iPadThirdGen4G, iPhone5, iPodTouchFifthGen, iPadFourthGen, iPadFourthGen4G, iPadMini, iPadMini4G, iPhone5c, iPhone5s, iPhone6, iPhone6Plus, iPodTouchSixthGen
**Original Artwork:**
<img src="http://is5.mzstatic.com/image/thumb/Purple30/v4/ab/99/a3/ab99a3a9-b1b1-0941-8a7e-efe517dff942/source/1024x1024bb.png" width="180" height="180" />
**Accepted Artwork:**
![](http://preview.dreamnet.at/?image=https://cloud.githubusercontent.com/assets/20930745/18688717/d46b7206-7f85-11e6-975f-e53db1ec7dff.png)

**Comment by 2Qt2beStr8** [comment]
![](http://preview.dreamnet.at/?image=https://cloud.githubusercontent.com/assets/20930745/18606836/5b927a68-7cbb-11e6-9302-8444a83c93dc.png)
https://cloud.githubusercontent.com/assets/20930745/18606836/5b927a68-7cbb-11e6-9302-8444a83c93dc.png

Source:
https://cloud.githubusercontent.com/assets/20930745/18606837/6bcb3712-7cbb-11e6-960a-7aed521dd414.png

**Comment by 2Qt2beStr8** [comment]
one more try ... but not really better :-( ... i don't like this icon :-)

![](http://preview.dreamnet.at/?image=https://cloud.githubusercontent.com/assets/20930745/18688717/d46b7206-7f85-11e6-975f-e53db1ec7dff.png)
https://cloud.githubusercontent.com/assets/20930745/18688717/d46b7206-7f85-11e6-975f-e53db1ec7dff.png

Source:
https://cloud.githubusercontent.com/assets/20930745/18688730/e9734976-7f85-11e6-9eab-f3a48611840f.png

**Comment by dreamnet** [comment]
I take the first one, because @GMVargas has reacted to it with thumbs up.

**Comment by GMVargas** [comment]
I hope it isn't too late but after seeing the 2nd icon, I feel like the 2nd one should be the official. I gave the 1st one a thumbs up because I thought it looked good enough and at the time it was the _only_ one uploaded.
I've uploaded 2 screenshots with both icons being used and the 2nd one looks much better. I also like how the flags are actual flags and not triangles, it looks much closer to the original.
![untitled2](https://cloud.githubusercontent.com/assets/9070923/18732025/cbb642aa-8014-11e6-89f1-3359040e4796.jpg)
![untitled](https://cloud.githubusercontent.com/assets/9070923/18732027/ce97c598-8014-11e6-8b86-f2c3d4d76293.jpg)
